src/components/sequencer/sequencer-step.tsx

```diff
@@ -61,10 +61,12 @@ const SequencerStep: React.FC<SequencerStepProps> = (
         [index, onChange, value]
     );
 
+    const isContentEmpty = !hasSamples && value.isEmpty();
+
     return (
         <ConditionalTooltip
             content="Select one or more samples to drop in"
-            shouldRender={!hasSamples && value.isEmpty()}>
+            isShown={isContentEmpty ? undefined : false}>
```

Comment:
I was thinking we could actually override the `isShown` prop via `ConditionalTooltip` - instead of being an optional boolean (`isShown?: boolean`), we could omit that from `TooltipProps` and instead mark it as required:

```tsx
interface ConditionalTooltipProps extends Omit<TooltipProps, 'isShown'> {
  isShown: boolean
}

// ... in render
<Tooltip isShown={props.isShown ? undefined : false } />
```

This provides a more binary API around the 'technically correct' but not-so-obvious behavior of `true`, `false`, and `undefined` for the base `Tooltip`. With this change, we could put weird ternary in one place - and not all over.

Reply:
Oh, I read ya. I can take that route.